### DISCUSSION TITLE
Custom extension error vs official

### DISCUSSION THREAD

**Discussion by Eaielectronic** [TurboWarp/extensions#1826] [Q&A]
Hi, I'd like to know how to do this because I've modified the Cloudlink extension and it works fine with my changes, but when I compile it, it retrieves the official TurboWarp version, so it doesn't work. How can I do this?

**Comment by yuri-kiss** [comment]
wdym compile it

**Comment by Drago-Cuven** [comment]
change the id.

**Reply by yuri-kiss** [reply]
turbowarp doesnt fetch extensions based on id
otherwise a lot of stuff would be broken

**Reply by yuri-kiss** [reply]
and cubester for future reference wait for more clarification before marking something as the answer please
